### DISCUSSION TITLE
Integration for MySQL

### DISCUSSION THREAD

**Discussion by denisgavioli** [JonPSmith/AuthPermissions.AspNetCore#86] [Ideas]
Hello Jon, have you any roadmap to support MySQL/MariaDB in AuthPermissions.AspNetCore?
Thank you,
Denis

**Comment by JonPSmith** [comment]
Hi @hcsrl,

I have just released version 5.0.0 of the AuthP library that contains the feature you need, called "custom databases. See the [Setup the custom database feature](https://github.com/JonPSmith/AuthPermissions.AspNetCore/wiki/Setup-the-custom-database-feature) for more information.

I have also written the [first article](https://www.thereformedprogrammer.net/using-custom-databases-with-the-authp-library-part1-normal-apps/) about this feature than might be useful too.

**Reply by denisgavioli** [reply]
Thank you very much! I'll take a look at it!
